pkg/extensions/config/sync/config.go

```diff
@@ -33,7 +33,9 @@ type RegistryConfig struct {
 	RetryDelay       *time.Duration
 	OnlySigned       *bool
 	CredentialHelper string
-	PreserveDigest   bool // sync without converting
+	PreserveDigest   bool     // sync without converting
+	Architectures    []string `mapstructure:",omitempty"` // filter architectures during sync (DEPRECATED: use Platforms instead)
```

Comment:
Would just drop Architectures field and keep only the Platforms field. Makes config too complex/verbose.

Realistically, this is a selector that narrows down choices.

Reply:
Sorted in https://github.com/project-zot/zot/pull/3128/commits/3eecc290ecd167becd0dec432d91c11b2e82c0cc